orangecontrib/timeseries/datasources.py

```diff
@@ -2,8 +2,11 @@
 from datetime import date
 from enum import Enum
 
+import pandas_datareader.data as web
+
 from Orange.data import Domain
 from orangecontrib.timeseries import Timeseries
+from orangecontrib.timeseries.pandas_compat import table_from_frame
 
 
 class DataGranularity(Enum):
```

Comment:
If unused, remove.

Reply:
Yes, it is unused. And method `quandl_data` is unused as well.